shopelectro/views/ecommerce.py

```diff
@@ -18,9 +20,13 @@ class OrderPage(ec_views.OrderPage):
     cart = SECart
 
     def get_context_data(self, request, **kwargs):
+        data = super().get_context_data(request, **kwargs)
         return {
-            **super(OrderPage, self).get_context_data(request, **kwargs),
+            **data,
             'page': CustomPage.objects.get(slug='order'),
+            'order_fields': json.dumps({
```

Comment:
`order_fields_html` might be the more good name.
Usually, we send to template data structures such as arrays or dicts.
But here we have plain string to render